test/functional/apps/discover/_field_data.js

```diff
@@ -0,0 +1,268 @@
+      var common;
+      var headerPage;
+      var settingsPage;
+      var discoverPage;
+
+      bdd.before(function () {
+        common = new Common(this.remote);
+        headerPage = new HeaderPage(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        discoverPage = new DiscoverPage(this.remote);
+        var fromTime = '2015-09-19 06:31:44.000';
+        var toTime = '2015-09-23 18:31:44.000';
+
+        // start each test with an empty kibana index
+        return scenarioManager.reload('emptyKibana')
+        // and load a set of makelogs data
+        .then(function loadIfEmptyMakelogs() {
+          return scenarioManager.loadIfEmpty('logstashFunctional');
+        })
+        .then(function (navigateTo) {
+          common.debug('navigateTo');
+          return settingsPage.navigateTo();
+        })
+        .then(function () {
+          common.debug('createIndexPattern');
+          return settingsPage.createIndexPattern();
+        })
+        .then(function () {
+          common.debug('discover');
+          return common.navigateToApp('discover');
+        })
+        .then(function () {
+          common.debug('setAbsoluteRange');
+          return headerPage.setAbsoluteRange(fromTime, toTime);
+        })
+        .catch(common.handleError(this));
+      });
+
+
+      bdd.describe('field data', function () {
+        var queryName1 = 'Query # 1';
+        var fromTimeString = 'September 19th 2015, 06:31:44.000';
+        var toTimeString = 'September 23rd 2015, 18:31:44.000';
+
+
+        bdd.it('search php should show the correct hit count', function () {
+          var expectedHitCount = '445';
+          return discoverPage.query('php')
+          .then(function () {
+            return common.tryForTime(20 * 1000, function tryingForTime() {
+              return discoverPage.getHitCount()
+              .then(function compareData(hitCount) {
+                expect(hitCount).to.be(expectedHitCount);
+              });
+            });
+          })
+          .catch(common.handleError(this));
+        });
+
+        bdd.it('the search term should be highlighted in the field data', function () {
+          // marks is the style that highlights the text in yellow
+          return discoverPage.getMarks()
+          .then(function (marks) {
+            expect(marks.length).to.be(50);
+            expect(marks.indexOf('php')).to.be(0);
+          })
+          .catch(common.handleError(this));
+        });
+
+
+        bdd.it('search _type:apache should show the correct hit count', function () {
+          var expectedHitCount = '11,156';
+          return discoverPage.query('_type:apache')
+          .then(function () {
+            return common.tryForTime(20 * 1000, function tryingForTime() {
+              return discoverPage.getHitCount()
+              .then(function compareData(hitCount) {
+                expect(hitCount).to.be(expectedHitCount);
```

Comment:
I'm not sure which block you mean.  Which line number?  Line 84 starts a tryForTime block so the getHitCount does need to be nested under that.